system-design/margo-api-reference/workload-api/desired-state-api/desired-state.md

```diff
@@ -0,0 +1,225 @@
+# Desired State
+
+> Action: This is incomplete and doesn't contain the details for the deploymentProfile or parameters
+
+The desired state is expressed as a [Kubernetes custom resource definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and made available to the device's management client as a YAML document using the OpenGitOps pattern.
```

Comment:
By simply referring to "using the OpenGitOps pattern", are we defining the API clearly enough to reach interoperability?

Reply:
I have changed the wording on the Action item and will resolve this conversation for now. This topic will be further discussed in future meetings.